jupyter/muehle-minimax.ipynb

```diff
@@ -0,0 +1,118 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Minimax"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%run ./muehle-game.ipynb\n",
+    "import sys\n",
+    "import inspect"
+   ]
+  },
+  {
+   "source": [
+    "memoize(f) takes a function and checks if the function was already called with the same two main arguments and returns the saved return value. If it is not yet in the Cache the function and it's return values will be saved to the Cache"
```

Comment:
Ich bin mir gerade nicht sicher: Wollten wir nicht die Beschreibungen auf Deutsch machen? Muss jetzt auch nicht in diesem PR sein, nur so generell.